### DISCUSSION TITLE
License key lost

### DISCUSSION THREAD

**Discussion by harlequin76** [aclap-dev/video-downloadhelper#621] [Help and Q&A]
### Your question

I didn't receive the license key (i bought today) by email and now I can't check it download helper. 
How can I get it again?
Thank you. 

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: it
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 26
}

**Comment by jcfrog** [comment]
hi,
please send an email at licenses@downloadhelper.net
